peps/pep-0772.rst

```diff
@@ -184,201 +184,196 @@ making process that governs the technical projects under the PyPA umbrella.
 
-Initial membership
-------------------
-
-Initial set of voting members will include anyone who has taken the time to
-formalise their participation in the packaging community. This includes:
-
-* PyPA members: Anyone with the triage bit or commit bit, or at least one
-  project in the PyPA organisation.
-* Packaging workgroup members: Anyone who is listed on the Packaging WG charter
-  who is willing to participate is welcome.
-* Interested core team members: Any Python core team member who is willing to
-  participate is welcome.
-* Wider community members: An initial set of for-profit companies, nonprofit
-  organizations, academic or educational institutions and smaller unaffiliated
-  projects would be invited to nominate three individuals to represent them.
-  See :ref:`the initial set of community voting members <appendix_C>`.
-
-Adding a new member
--------------------
-
-New voting members are added by a simple majority vote by the current
-membership. Quorum for adding new members is 50%.
-
-A vote to add a new member is triggered when a voting member calls for one
-publicly on an appropriate communication channel, and another voting member
-seconds the call within two weeks.
-
-The vote lasts for one week. Each member can vote for or against.
-
 Removal of a member
 -------------------
 
-In order to maintain a reasonable expectation of quorum, failure to participate
-in Packaging Council elections for two consecutive council elections
-automatically removes a person from the list of voting members, until they
-re-submit their intention to resume their participation to the Packaging
-Council in writing.
+In order to maintain a reasonable expectation of quorum, failure to participate in Packaging Council elections
+for two consecutive Council elections automatically removes a person from the list of Packaging Council
+Electors, until they re-submit their intention to resume their participation to the Packaging Council in
+writing.
 
-In exceptional circumstances, it may be necessary to remove someone from the
-voting members against their will (for example: egregious and ongoing code of
-conduct violations). A voting member may be removed by a two-thirds majority
-vote by the Packaging Council (in practice: 4:1 for a council with five
-members).
+In exceptional circumstances, it may be necessary to remove someone from the Electors against their will (for
+example: egregious and ongoing code of conduct violations). An Elector may be removed by a two-thirds majority
+vote by the Packaging Council (in practice: 4:1 for a Council with five members).
 
-If the relevant voting member is also on the Packaging Council, then they can
-participate in the vote. They are removed from the Packaging Council if the
-vote removes them as a voting member. The vacancy is filled as per the process
-for filling vacancies in the Packaging Council.
+If the relevant Elector is also on the Packaging Council, then they can participate in the vote. They are
```

Comment:
If the individual removed from the Electors roll is also on the Packaging Council, they lose the privilege to vote in future elections. The seated Packaging Council may vote to remove the individual from the Council, and the individual must abstain from a removal vote. The vacancy ...

Reply:
I don't follow this. (An individual removed from the Electors roll always loses the privilege to vote in future elections, regardless of who that individual is, because that's what it means to be removed. Also the Packaging Council was the body that voted to remove them as an Elector; why vote twice, and possibly differently, to keep them on the Council?)

Reply:
I think this thread can also be resolved, given the commented text is outdated.